src/main/java/teammates/ui/controller/InstructorFeedbackAbstractAction.java

```diff
@@ -0,0 +1,147 @@
+            newSession.setSessionVisibleFromTime(TimeHelper.combineDateTime(
+                    getRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_VISIBLEDATE),
+                    getRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_VISIBLETIME)));
+            break;
+        case Const.INSTRUCTOR_FEEDBACK_SESSION_VISIBLE_TIME_ATOPEN:
+            newSession.setSessionVisibleFromTime(Const.TIME_REPRESENTS_FOLLOW_OPENING);
+            break;
+        case Const.INSTRUCTOR_FEEDBACK_SESSION_VISIBLE_TIME_NEVER:
+            newSession.setSessionVisibleFromTime(Const.TIME_REPRESENTS_NEVER);
+            // overwrite if private
+            newSession.setResultsVisibleFromTime(Const.TIME_REPRESENTS_NEVER);
+            newSession.setFeedbackSessionType(FeedbackSessionType.PRIVATE);
+            break;
+        default:
+            log.severe("Invalid sessionVisibleFrom setting in creating "
+                       + newSession.getIdentificationString());
+            break;
+        }
+
+        String[] sendReminderEmailsArray =
+                getRequestParamValues(Const.ParamsNames.FEEDBACK_SESSION_SENDREMINDEREMAIL);
+        List<String> sendReminderEmailsList =
+                sendReminderEmailsArray == null ? new ArrayList<String>()
+                                                : Arrays.asList(sendReminderEmailsArray);
+        newSession.setClosingEmailEnabled(
+                sendReminderEmailsList.contains(EmailType.FEEDBACK_CLOSING.toString()));
+        newSession.setPublishedEmailEnabled(
+                sendReminderEmailsList.contains(EmailType.FEEDBACK_PUBLISHED.toString()));
+        return newSession;
+    }
+
+    protected List<FeedbackSessionAttributes> loadFeedbackSessionsList(
+            List<InstructorAttributes> instructorList) {
+
+        List<FeedbackSessionAttributes> sessions =
+                logic.getFeedbackSessionsListForInstructor(instructorList);
+        return sessions;
+    }
+
+    protected List<CourseAttributes> loadCoursesList(List<InstructorAttributes> instructorList) {
+
+        List<CourseAttributes> courses = logic.getCoursesForInstructor(instructorList);
+
+        Collections.sort(courses, new Comparator<CourseAttributes>() {
+            @Override
+            public int compare(CourseAttributes c1, CourseAttributes c2) {
+                return c1.getId().compareTo(c2.getId());
+            }
+        });
+
+        return courses;
+    }
+
+    /**
+     * Gets a Map with courseId as key, and InstructorAttributes as value.
+     */
+    protected Map<String, InstructorAttributes> loadCourseInstructorMap(boolean omitArchived) {
+        Map<String, InstructorAttributes> courseInstructorMap = new HashMap<String, InstructorAttributes>();
+        List<InstructorAttributes> instructors = logic.getInstructorsForGoogleId(account.googleId,
+                                                                                 omitArchived);
```

Comment:
We've decided to move away from custom indentation. Use 8 spaces but insert the line break at as high level as possible.